setup.py

```diff
@@ -23,12 +23,15 @@
     namespace_packages = ['zc'],
     package_dir = {'': 'src'},
     install_requires = ['setuptools',
-                        'ZODB3 >=3.9dev'
+                        'ZODB<5',
+                        'ZEO<5',
+                        'transaction<2',
                         ],
-    extras_require=dict(
-        test=[
-            'zope.testing',
-            ]),
+    tests_require=[
+        'zope.testing',
+        'mock',
+    ],
```

Comment:
I would prefer if we kept the `[test]` extra: there's overall movement in zopefoundation/* to ditch `python setup.py test` and start using zope-testrunner directly, for which installing the tests with `pip install -e .[test]` or via tox's `extras = test` is very helpful.

Reply:
I would like to use `zope-testrunner` in another pull request. What do you think?

Reply:
I think it's fine to defer changing the testrunner to a later PR. But the `test` extra needs to stay (be returned) in this PR.

Reply:
Sure, that was a description of the general trends, not a requirement for this PR.

Reply:
I restored `test` extra, but I also kept `test_require` for now so `python setup.py test` still work.